geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -5613,7 +5613,7 @@ public void cleanupFailedInitialization() {
     }
     if (savedFirstRuntimeException != null
         && savedFirstRuntimeException instanceof DistributedSystemDisconnectedException) {
-      logger.warn("cleanupFailedInitialization originally failed with {}",
+      logger.warn("cleanupFailedInitialization originally failed with: ",
```

Comment:
Did the old code for this warning log the call stack for savedFirstRuntimeException? It looks like the intent was just to log savedFirstRuntimeException.toString() inside the curly brackets. 
But if the old behavior was to log the stack then I think your change is good.

All the places we consider setting savedFirstRuntimeException we also log a warning with the full stack. So I think that intent of this message was just to say "cleanup actually did fail and here is the exception of all the ones we already logged that caused the failure". It seems odd to me that these are warnings. I'm not sure what a customer could do with these warnings.

Reply:
I like having the stack on warnings.